### PR TITLE
fix(kuma-cp) set cluster ID to followers

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

This PR fixes two problems
* Report proper number of zones. We were reporting meshes as zones and only when zones > 0. We want to report zones as 1 on standalone and real number of zones on global
* Cluster ID was only set on leader on standalone and global. I refactored the code so we set the cluster ID in consistent way in all use cases.

### Documentation

- [X] No docs

### CODE REVIEW

File: pkg/clusterid/creator.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	return c.create()
+}
+
+func (c *clusterIDCreator) NeedLeaderElection() bool {
+	return true
+}
+
+func (c *clusterIDCreator) create() error {
+	resource := config_model.NewConfigResource()
+	err := c.configManager.Get(context.Background(), resource, store.GetByKey(config_manager.ClusterIdConfigKey, ""))
```
Comment (lobkovilya) [inline pkg/clusterid/creator.go:28]: I think empty string should be replaced with `model.NoMesh`

File: pkg/clusterid/creator.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	if err != nil {
+		if !store.IsResourceNotFound(err) {
+			return err
+		}
+		resource.Spec.Config = core.NewUUID()
+		log.Info("creating cluster ID", "clusterID", resource.Spec.Config)
+		if err := c.configManager.Create(context.Background(), resource, store.CreateByKey(config_manager.ClusterIdConfigKey, "")); err != nil {
```
Comment (lobkovilya) [inline pkg/clusterid/creator.go:35]: same here

File: pkg/clusterid/cluster_id_test.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		// when runtime is started
+		stop = make(chan struct{})
+		go func() {
+			defer GinkgoRecover()
+			err := runtime.Start(stop)
+			Expect(err).ToNot(HaveOccurred())
+		}()
+
+		// then cluster ID is created and set in Runtime object
+		Eventually(runtime.GetClusterId, "5s", "100ms").ShouldNot(BeEmpty())
```
Comment (lobkovilya) [inline pkg/clusterid/cluster_id_test.go:42]: I know it was the same before your changes, but is it okay that we can dispatch empty `cluster_id`?

Comment (jakubdyszkiewicz) [reply pkg/clusterid/cluster_id_test.go:42]: yes, it is ok.

**Review by lobkovilya — APPROVED** [review]

**Pull request merged**